chronicle.xcodeproj/project.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>PreviewsEnabled</key>
+	<false/>
+</dict>
+</plist>
```

Comment:
What does this do? Wondering if we really need this :)

Reply:
good question, i'm not sure honestly—it was automatically generated at some point when i first started adding code. i kinda think it was generated when i added the dark mode preview. the thing is, i'm not sure if you can delete this?

Reply:
@giuliacampana  can you try deleting this locally and verify if the project compiles?

Reply:
actually that's the thing... the local file seems to be unfindable, as though there isn't one

Reply:
yeah, that's weird. Anyway, I don't think it matters. Marking this as resolved